JNG-3324: osgi-configuration-mapper

kristofkakuszi opened the pull request.

kristofkakuszi committed:
JNG-3324: osgi-configuration-mapper

kristofkakuszi committed:
JNG-3324: osgi-configuration-mapper
execute permission for mvnw added

noherczeg committed:
JNG-3324: remove buildkite config

kristofkakuszi committed:
JNG-3324: osgi-configuration-mapper
execute permission for mvnw added
sign-artifacts profile changed

noherczeg commented on pom.xml:
```diff
@@ -202,38 +231,53 @@
-             -->
-            
+
+            <plugin>                                                                <!--added till the end-->
```
We should loose this comment.

kristofkakuszi replied:
Done. Deleted unnecessary comments

kristofkakuszi committed:
JNG-3324: osgi-configuration-mapper
execute permission for mvnw added
sign-artifacts profile changed
Deleted unnecessary comments
